### DISCUSSION TITLE
Message d'erreur, mais ça fonctionne

### DISCUSSION THREAD

**Discussion by jodionrq** [MTESSDev/FRW#61] [Annonces]
Bonjour,
Dans VSCode j'ai un message d'erreur soulignée en rouge pour une commande de validation de date. Mais dans le bac-à-sable, tout semble fonctionner parfaitement. Voir capture d'écran. Le message d'erreur apparait comme attendu. 
![image](https://user-images.githubusercontent.com/124822406/232897931-21e7439a-9990-4c98-8dfd-24ad7fd21d6a.png)

Si j'entre l'information comme il faut, tout fonctionne aussi
![image](https://user-images.githubusercontent.com/124822406/232898296-4aa93d41-665b-46b7-bb48-235f10bdfcfd.png)

Est-ce que ça peut causer problème? Est-ce qu'il y a une autre commande à utiliser? Merci!

**Comment by anisite** [comment]
Bonsoir, il s'agit effectivement d'un bug, j'ouvre un issue, Il s'agit simplement d'une erreur avec le schéma de validation, nous allons l'ajuster, ça ne causera pas d'autre problèmes, ça peut rester comme ça en attendant.

Merci de l'information.

**Reply by jodionrq** [reply]
Bonjour, merci beaucoup pour le suivi. Nous allons aller de l'avant en ne tenant pas compte de ce bug.
Bonne journée!

**Comment by Hutson48** [comment]
Bug réglé
